packages/generator/src/generate.ts

```diff
@@ -62,7 +55,9 @@ const generate = async (props: GenerateProps) => {
           continue;
         }
 
-        const render = rendererRegistry[schema.type];
+        const render = Object.values(plugins).find(
+          (plugin) => plugin.propPanel.defaultSchema.type === schema.type
```

Comment:
There may be users who are only using the generator, but accessing `plugin.propPanel.defaultSchema.type` feels a bit awkward. Should it be accessible through `plugin.type` instead?

Reply:
I might be overthinking this. If it doesn't feel awkward as is, I'd like to keep it that way.